etna/records/field_labels.py

```diff
@@ -1,12 +1,12 @@
 # format <record_field_name> : <human readable label name>
 FIELD_LABELS = {
-    "title": "Title",
+    "summary_title": "Summary title",
```

Comment:
Since the records have both, maybe we should keep both here?

Reply:
Bringing in Title needs approval https://national-archives.atlassian.net/browse/DF-417